tests/checks/mock/test_rabbitmq.py

```diff
@@ -0,0 +1,62 @@
+import sys
+
+import mock
+import requests
+
+from checks import AgentCheck
+from tests.checks.common import AgentCheckTest, get_checksd_path, get_os
+
+
+class TestRabbitMQ(AgentCheckTest):
+
+    CHECK_NAME = 'rabbitmq'
+
+    @classmethod
+    def setUpClass(cls):
+        sys.path.append(get_checksd_path(get_os()))
+
+    @classmethod
+    def tearDownClass(cls):
+        sys.path.pop()
+
+    def test__get_data(self):
+        with mock.patch('rabbitmq.requests') as r:
+            from rabbitmq import RabbitMQ, RabbitMQException  # pylint: disable=import-error
+            check = RabbitMQ('rabbitmq', {}, {"instances": [{"rabbitmq_api_url": "http://example.com"}]})
+            r.get.side_effect = [requests.exceptions.HTTPError, ValueError]
+            self.assertRaises(RabbitMQException, check._get_data, '')
+            self.assertRaises(RabbitMQException, check._get_data, '')
```

Comment:
It's to consume both the `side_effect`s so we test it both with `HTTPError` (or any requests exception) and `ValueError`